api/v1/helper_test.go

```diff
@@ -77,7 +78,8 @@ func newNodePolicy() *v1.SriovNetworkNodePolicy {
 			Name: "p1",
 		},
 		Spec: v1.SriovNetworkNodePolicySpec{
-			DeviceType: "netdevice",
+			DeviceType: consts.DeviceTypeNetDevice,
+			VdpaType:   consts.VdpaTypeVirtio,
```

Comment:
can you please have a newPolicy for the vdpa test?
I will like to validate that we continue to support policies without VdpaType configured

Reply:
It makes sense, will do.